framework/src/main/java/org/checkerframework/framework/type/visitor/README

```diff
@@ -0,0 +1,2 @@
+This directory does not just contain visitors.  It contains types that are
+intended to be subclassed, such as interfaces and abstract classes.
```

Comment:
` AnnotatedTypeScanner`, ` SimpleAnnotatedTypeScanner`, and `SimpleAnnotatedTypeVisitor` currently aren't abstract. They should be, right?